src/Card/Card.scss

```diff
@@ -422,6 +422,8 @@ a .pgn__card {
   border-radius: 0 0 .375rem .375rem;
```

Comment:
It looks like the padding used for `Card.Status` is a bit more than the padding used for the other `Card` subcomponents like `Card.Footer` and `Card.Section` (1.5rem vs. 1.25rem respectively). Should the padding above (i.e., `padding: 1.5rem;`) be using the `$card-spacer-x` and/or `$card-spacer-y` variables to ensure the padding is consistent?

Reply:
I tested with `$card-spacer-x` and `$card-spacer-y` and it looks good. Worth noting that it is important to hold the logic behind spacings and sizes in the component especially in such big as Card, Datatable etc. Then it is easier to extend and modify for Paragon users.